CuriousKatie/CuriousKatie/Model/Person.swift

```diff
@@ -14,28 +14,54 @@ struct Person {
     
     var name: String
     var age: Int
-    var interests = CuriousKatie.pickInterests()
+    var interests = pickInterests()
     
 
     func introduce() -> String {
        
-        let greeting = ["Hello everybody, I'm ", "Hi all, my name is", "Hey there, they call me"]
-        let introduction = "\(greeting.randomElement()!) \(name), and I'm \(age) years old."
+        let greetings = ["Hello everybody, I'm ", "Hi all, my name is", "Hey there, they call me"]
+        let introduction = "\(greetings.randomElement()!) \(name), and I'm \(age) years old."
     
         return introduction
     }
     
-    func shareInterest(indexPath: Int) -> String {
-        let sharedInterestTitle = interests[indexPath].title
-        let sharedInterestSkill = interests[indexPath].skill
-        
-        return "(\(name))My interest is " + sharedInterestTitle + ", I'm " + sharedInterestSkill.rawValue
-        
+    func shareInterest(indexPath: Int) -> Any {
+        if indexPath <= interests.count - 1{
+            let interest = interests[indexPath]
+            return name + "My interest is " + interest.title + ", I'm " + interest.skill.rawValue
+        } else {
+            return name + "All interests said"
+        }
     }
     
 
     init(name: String, age: Int) {
         self.name = name
         self.age = age
     }
+    
+    static func generateParticipants() -> [Person] {
```

Comment:
To when making an explicit (does not depend on any dynamic/parameter value that needs to be passed in) computation,
- Computed properties are better.
```suggestion
    static var participants: [Person] {
```
Additionally, because **participants** seems to be more related to the definition of CuriousKatie, this should probably have a name that fits the context of **Person** more...
```suggestion
    static var people: [Person] {
```